### DISCUSSION TITLE
FAQ/Guide Requests

### DISCUSSION THREAD

**Discussion by msprijatelj** [massif-press/compcon#2059] [Archive]
If there is an aspect of COMP/CON that you feel would benefit from a guide or FAQ entry within the [Wiki’s FAQ](https://github.com/massif-press/compcon/wiki/Frequently-Asked-Questions), please feel free to mention it in this discussion! Try to keep to the following format:

### Reason for entry
What action in COMP/CON is unclear or difficult to manage, and would benefit from a guide?

### Desired information
What information should the entry provide?

### Additional Context
Any other relevant information.

**Comment by pkarjala** [comment]
I have gone through and done a general clean up of the FAQ content in the wiki, and have started creating a new guide section.

As a note, the Troubleshooting page was moved to its own section at https://github.com/massif-press/compcon/wiki/Troubleshooting-Issues.  The link from the Help page on the COMP/CON application will need to be updated to reflect this change.  I will create an issue for this and, if time allows, make a pull request.

**Reply by msprijatelj** [reply]
PR created to address the issue! https://github.com/massif-press/compcon/pull/2111
